lib/linguist/lazy_blob.rb

```diff
@@ -24,6 +24,10 @@ def initialize(repo, oid, name, mode = nil)
       @mode = mode
     end
 
+    def path
+      name
+    end
```

Comment:
Hm. I wonder if we should rename `@name` to `@path` then, and if we should make `#name` just return the baseman.